WooCommerce/WooCommerceUITests/Screens/Login/LoginEpilogueScreen.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+import XCTest
+
+private struct ElementStringIDs {
+    static let displayNameField = "full-name-label"
+    static let siteUrlField = "url-label"
+    static let continueButton = "login-epilogue-continue-button"
+}
+
+class LoginEpilogueScreen: BaseScreen {
+    let continueButton: XCUIElement
+    let displayNameField: XCUIElement
+    let siteUrlField: XCUIElement
```

Comment:
This comment is applicable to all the `Screen` classes in this PR.

I like that we _force_ call sites to use the functions like `continueWithSelectedSite()`. I believe we can even be more explicit by making these `private` too. 

Making these properties public could give future developers (i.e. me) a false signal that they can use these properties directly. Unless that is the intent?

```suggestion
    private let continueButton: XCUIElement
    private let displayNameField: XCUIElement
    private let siteUrlField: XCUIElement
```

Reply:
Good point! At this point our UI tests are focused on functional testing of flows through the app (rather than e.g. exhaustively testing every element on a screen) so I think it makes sense to make these properties private, to enforce using the methods for each page object.

I could imagine making these properties public if we decide to diversify the kinds of UI tests we have, but it'd be better to make that an explicit change in how we do things.